src/thruster.cpp

```diff
@@ -79,6 +79,12 @@ void Thruster::Load(physics::ModelPtr _parent, sdf::ElementPtr _sdf)
     }
     ROS_DEBUG_STREAM("back_thrust_ratio: " << back_thrust_ratio);
 
+    if(!ros::param::get("visualize_thrusters", visualize_thrusters))
+    {
+        ROS_WARN_STREAM("failed to load thruster visualizer on/off. defaulting to on");
+        visualize_thrusters = true;
```

Comment:
This is only loaded here on initialization, and so cannot be changed while the simulator is running. Also, because the value defaults to true and since the true default is loaded from the param file, this cannot be altered without changing the param file. Is this intended behavior?